frontend/packages/dev-console/src/utils/__tests__/pipeline-utils.spec.ts

```diff
@@ -16,4 +22,14 @@ describe('pipeline-utils ', () => {
     expect(stages[2]).toHaveLength(2);
     expect(stages[3]).toHaveLength(1);
   });
+  it('should return correct Container Status', () => {
+    let status = containerToLogSourceStatus(mockContainerStatus[0]);
```

Comment:
IMO unless the data set is large, it's more appealing to have the data structure inline so it's more clear to a reader what is being tested.

Reply:
^^ @abhinandan13jan